build/buildBuildImages.sh

```diff
@@ -409,6 +418,7 @@ function buildBuildPackImage() {
 }
 
 if [ -z "$imageTypeToBuild" ]; then
+	buildGitHubActionsImage "bullseye"
```

Comment:
Are we only creating a bullseye-based image for the GitHub Actions image right now? (_i.e._, ignoring Jamstack, LTS, etc.)

Reply:
Thanks for bringing it up, for this PR we only supported the github-actions bullseye based image. Not sure if Jamstack or lts-version image are also needed from App Service, will follow up with them regarding it.

Reply:
Makes sense -- thanks for clarifying 😃